### PR TITLE
A4A: Implement the FAQs section on the new Migrations page.

### PR DESCRIPTION
**Pull request by jkguidaven**
This PR adds the FAQs section to the new Migrations page. **Please note that not all FAQs are included. I will update this later when we have the complete copies.**

Screenshot 2024-11-04 at 6 19 38 PM

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1380

## Proposed Changes

* Implement the FAQs section using the `FoldableFAQ` component.

## Why are these changes being made?

* This is part of the new Migrations page project.

## Testing Instructions

* Use the A4A live link and go to the `/migrations` page.
* Confirm you see the FAQs section with a set of questions.
* Confirm that expanding the question will add a hash fragment to the URL.
* Confirm that the hash fragment works correctly when refreshing the page.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/a8c-for-agencies/sections/migrations/migrations-overview-v2/sections/migrations-faqs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+	);
+
+	return (
+		<PageSection
+			heading={ translate( 'Frequently asked questions' ) }
+			description={ translate( 'Curious about the details? We’ve got answers.' ) }
+		>
+			<ul className="migrations-faqs">
+				<li>
+					<FoldableFAQ
```
Comment (yashwin) [inline client/a8c-for-agencies/sections/migrations/migrations-overview-v2/sections/migrations-faqs/index.tsx]: [NITPICK] 

I think it's better to keep all the FAQs in a constant and map them through it to display them. What do you think?

**Review by yashwin — COMMENTED** [review]
Looks good overall.

The font size is bigger here (24px). It should be 20px as per the design.

Screenshot 2024-11-04 at 3 21 53 PM

File: client/a8c-for-agencies/sections/migrations/migrations-overview-v2/sections/migrations-faqs/index.tsx (hunk 1 above)
Comment (jkguidaven) [reply client/a8c-for-agencies/sections/migrations/migrations-overview-v2/sections/migrations-faqs/index.tsx]: Good suggestion. this has been applied 👍

**Comment by jkguidaven** [comment]
> Looks good overall.
> 
> The font size is bigger here (24px). It should be 20px as per the design.
> 
> Screenshot 2024-11-04 at 3 21 53 PM

Good catch. This has been fixed. 👍

File: client/a8c-for-agencies/sections/migrations/migrations-overview-v2/sections/migrations-faqs/index.tsx [hunk 2]
```diff
@@ -48,150 +48,116 @@ export default function MigrationsFAQs() {
...
+			question: translate( 'Does my site already have to be hosted on WordPress?' ),
+			answer: translate(
+				'No. It doesn’t matter what CMS your site currently uses. You will be eligible for our migration offer if you migrate your site to WordPress.com or Pressable. However, we won’t be able to provide you with migration assistance in this case.'
+			),
+		},
+		{
+			id: 'migration-payment-process',
+			question: translate( 'How do I get paid?' ),
+			answer: translate(
+				'Once your sites have been active on WordPress.com or Pressable hosting for over 6 months, we will pay you the site migration commission during the payout windows listed {{PayoutKBLink}}here{{/PayoutKBLink}}. You will also need to add your payment details to our payment partner, Tipalti, which you can do {{paymentSettingLink}}here{{/paymentSettingLink}}.',
```
Comment (yashwin) [inline client/a8c-for-agencies/sections/migrations/migrations-overview-v2/sections/migrations-faqs/index.tsx]: [NITPICK]

Sorry for not noticing this earlier. Feel free to ignore this.

I think it is better to keep the component name consistent here.

```suggestion
				'Once your sites have been active on WordPress.com or Pressable hosting for over 6 months, we will pay you the site migration commission during the payout windows listed {{PayoutKBLink}}here{{/PayoutKBLink}}. You will also need to add your payment details to our payment partner, Tipalti, which you can do {{PaymentSettingLink}}here{{/PaymentSettingLink}}.',
```

**Review by yashwin — APPROVED** [review]
Thanks for addressing my comment. LGTM now 👍

File: client/a8c-for-agencies/sections/migrations/migrations-overview-v2/sections/migrations-faqs/index.tsx (hunk 2 above)
Comment (jkguidaven) [reply client/a8c-for-agencies/sections/migrations/migrations-overview-v2/sections/migrations-faqs/index.tsx]: Good catch, yes. This should be Capitalize 👍

**Review by yashwin — APPROVED** [review]
Thanks for addressing my comment. LGTM!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16984060

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @jkguidaven for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16984060).

**Pull request merged**